libraries/ae-inventory-processor/src/main/java/org/metaeffekt/core/inventory/processor/model/Constants.java

```diff
@@ -68,14 +68,16 @@ public final class Constants {
     public static final String KEY_DESCRIPTION = "Description";
     public static final String KEY_ARCHITECTURE = "Architecture";
     public static final String KEY_TYPE = "Type";
+    public static final String KEY_COMPONENT_SOURCE_TYPE = "Component Source Type";
     public static final String KEY_SOURCE_PROJECT = "Source Project";
 
     public static final String KEY_CONTAINER = "Container";
     public static final String KEY_ISSUE = "Issue";
 
     public static final String ARTIFACT_TYPE_PACKAGE = "package";
+    public static final String ARTIFACT_TYPE_MODULE = "module";
     public static final String ARTIFACT_TYPE_FILE = "file";
-    public static final String ARTIFACT_TYPE_NODEJS_MODULE = "nodejs-module";
+    public static final String ARTIFACT_TYPE_WEB_MODULE = "web-module";
```

Comment:
The change is ok; we only need to check backwards compatibility with existing inventories.

Reply:
Also check other occurrences in of the string nodejs-module in core. There are two more locations to adjust.

Reply:
We should keep the old constant and deprecate it.